test/browser/spec/ol/interaction/keyboardpan.test.js

```diff
@@ -31,22 +31,22 @@ describe('ol.interaction.KeyboardPan', function () {
         preventDefault: Event.prototype.preventDefault,
       });
 
-      event.originalEvent.keyCode = 40; // DOWN
+      event.originalEvent.key = 'ArrowDown';
```

Comment:
Can we import and use `Key.DOWN` here?  Same below.

Reply:
Is that normal for tests?  If so should we not also be using `KEYDOWN`, etc. from `ol/events/EventType` in tests?  (A few tests do use `EventType.CHANGE` - in one case inconsistently using both that and the string - but most do not).

Reply:
Good point.  Small screen and I didn’t pay attention to which file this was in.